scripts/reboot

```diff
@@ -14,6 +14,10 @@ function stop_sonic_services()
         echo "Stopping syncd process..."
         docker exec -i syncd /usr/bin/syncd_request_shutdown --cold > /dev/null
         sleep 3
+
+        # Stop docker service to avoid 5-10 minutes delay in shutdown
+        # while waiting for existing docker containers to quit.
+        systemctl stop docker
```

Comment:
Should we put this "systemctl stop docker"  outside of the "if" statement so it is common for all platforms?

Reply:
Sharp eyes!

To add more protection, I also give this command a 30 seconds timeout.

Reply:
I assume 30 seconds would be more than enough for regular docker service stop.

Reply:
Correct. Normally stopping docker service takes about 15 seconds.